frontend/src/composables/useRequestTargetInputValidations.ts

```diff
@@ -27,6 +32,10 @@ export const useRequestTargetInputValidations = (
       selectedTargetAddress: {
         required: helpers.withMessage('Target address is required', required),
         isValidEthAddress: helpers.withMessage('Invalid ETH address', isValidEthAddress),
+        makeIsNotBlacklistedEthAddressValidator: helpers.withMessage(
```

Comment:
Is it necessary to have this twice? The dialog won't be loaded anyway and a user would not be able to create a transfer. Or am I missing something?

Reply:
This is for the cases where a user comes with a non-blocked wallet address and tries to bridge funds to the blocked address. I believe that these cases fall under the scope of suspicious activity as well and therefore i implemented this for extra safety.

Reply:
Ah, true. Didn't get it right. Now it makes sense to me :)